backend/resources/hacknight.py

```diff
@@ -15,7 +15,7 @@
 class HacknightList(Resource):
     @jwt_required
     def get(self):
-        hacknight_schema = HacknightSchema(many=True, exclude=("participants",))
+        hacknight_schema = HacknightSchema(many=True)
```

Comment:
Most likely it is topic for another ticket, but I am bit concern how big response might become after allowing to parse all participants for all hacknights. We will end up with bunch of repeated data (most likely these same participant object will be nested multiple times in different hacknight objects), which we don't really need - right now we are interested in number of participants only.